### PR TITLE
[Docs] Command Syntax and Parameter fixes - 4

### PR DESCRIPTION
**Pull request by seaniepie**

### CODE REVIEW

**Comment by seaniepie** [comment]
@montegoulding , This is the last of the command fixes for TravisCL. There's just a mammoth amount of 'property' items and a few 'functions' to go. Doing functions first

File: docs/dictionary/command/resolve-image.lcdoc [hunk 1]
```diff
@@ -18,6 +18,13 @@ Platforms: desktop, server, mobile
 Example:
 resolve image id 1007 relative to button "Cancel"
 
+Parameters:
+id or name:
+The short id or name of an image.
+
+object reference:
+The relative object to the image.
+
```
Comment (livecodeali) [inline docs/dictionary/command/resolve-image.lcdoc:27]: Could you make these parameters camel case please?

**Review by livecodeali — CHANGES_REQUESTED** [review]

File: docs/dictionary/command/resolve-image.lcdoc (hunk 1 above)
Comment (seaniepie) [reply docs/dictionary/command/resolve-image.lcdoc:27]: Done. 👍

**Comment by seaniepie** [comment]
Done 👍

File: docs/dictionary/command/resolve-image.lcdoc (hunk 1 above)
Comment (livecodeali) [reply docs/dictionary/command/resolve-image.lcdoc:27]: Sorry, can you do `idOrName` as well?

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 66283e5

**Review by livecodeali — APPROVED** [review]

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 66283e5d397f1588dd2e96a6dc9fdef1f83b5480

**Comment by livecode-vulcan** [comment]
😎 test success 66283e5d397f1588dd2e96a6dc9fdef1f83b5480
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/462)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/467)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/462)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/462)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/468)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/462)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/462)

**Pull request merged**